fastapi-google-stt/app/api/openai_router.py

```
@@ -41,5 +43,36 @@ async def get_generated_messages_with_header(
-        get_streaming_message_from_openai(data), media_type="text/event-stream"
+        get_streaming_message_from_openai(data), 
+        media_type="text/event-stream"
     )
+
+@router.post(
+    "/ask",
+    summary="Generate messages using OpenAI (streaming)",
+)
+async def ask_about_script(
+    data: AskRequest,
+    db: AsyncSession = Depends(get_db),
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)